.github/workflows/statistician-image.yml

```diff
@@ -50,21 +57,23 @@ jobs:
       run: |
         echo "VERSION=$(cat statistician/version.txt)" >> $GITHUB_ENV
 
-    - name: Build and Push unstable Docker Image from PR / Push to Master
-      uses: whoan/docker-build-with-cache-action@v4
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
       with:
-        context: statistician
-        image_name: ${{ env.IMAGE_NAME }}
-        username: ${{ secrets.DOCKER_USER }}
-        password: ${{ secrets.DOCKER_PASS }}
-        image_tag: latest,${{ env.VERSION }}
+        role-to-assume: arn:aws:iam::538673716275:role/github-ecr
```

Comment:
put this in secrets,

Reply:
I don't think this is necessary, see longer comment above.